recipe/meta.yaml

```
@@ -64,7 +64,7 @@ outputs:
         - m2-patch  # [win]
       host:
         - python
-        - pip
+        - pip <24.2
```

```suggestion
        - pip
```
This issue is only affecting pip check, no?

---

right!
removed